platform-ajax/src/main/java/com/softicar/platform/ajax/request/AjaxRequestMessageFetcher.java

```diff
@@ -0,0 +1,59 @@
+package com.softicar.platform.ajax.request;
+
+import com.softicar.platform.common.core.exceptions.SofticarIOException;
+import com.softicar.platform.common.io.HexDecoderStream;
+import com.softicar.platform.common.io.StreamUtils;
+import com.softicar.platform.common.string.charset.Charsets;
+import java.io.IOException;
+import java.util.Optional;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+
+class AjaxRequestMessageFetcher {
+
+	private static final String FORM_REQUEST_INPUT_NAME = "form-request-input";
+	private final HttpServletRequest request;
+	private final String contentType;
+
+	AjaxRequestMessageFetcher(HttpServletRequest request) {
```

Comment:
This CTOR could as well be `public` because the class is already default-visible.

Reply:
I'd rather argue that restricting method/CTOR visibility to match the class visibility is an unnecessary effort, since it's dominated by class visibility anyway.

Reply:
well, it is more code, on the other hand, with public it is easier to toggle the class between package visible and public

Reply:
changed to public